### DISCUSSION TITLE
Cloning a kart postgis repository

### DISCUSSION THREAD

**Discussion by marcathian** [koordinates/kart#720] [Q&A]
Good day
I have been able to create a kart repository on a postgis database.
I now wish to clone this repository on my workstations.
How do I clone the repository?

I am unable to successfully clone the repo, both with the QGIS plugin and the command line

**on the command line I have tried:**
kart clone host[:port]/dbname/dbschema
I got the following error:

Cloning into 'C:\LRTP2GIS\Kartexplore\trycln1\trykart\.clone'...
warning: templates not found in C:\Program Files\Kart\share\git-core\templates
Host key verification failed.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
Error: Error calling git clone

**I have also tried on the command line:**
kart clone postgresql://[user[:password]@]host[:port]/dbname/dbschema
I got the following error:
Cloning into 'C:\LRTP2GIS\Kartexplore\trycln1\trykart\.clone'...
warning: templates not found in C:\Program Files\Kart\share\git-core\templates
git: 'remote-postgresql' is not a git command. See 'git --help'.
Error: Error calling git clone

**Comment by olsen232** [comment]
Hi! Thanks for trying out Kart, sorry for the delayed response.

The entity which can be cloned is a Kart repository, which is stored on your filesystem. It is very similar to a Git repository. That is what contains all of the branches and commits - the various revisions of your data.
Kart supports a few different "working copy" formats. A working copy is where you check out a particular revision of your data, which you can then view or edit and commit changes as a new revision. The working copy cannot be cloned, but if you clone the repository, you will then be able to make a new working copy that's connected to your new repository.
Let me know if you need help finding the folder on your machine that contains the Kart repository.

You can clone from one repository to another on the same machine using the following syntax:
`kart clone PATH-TO-SOURCE PATH-TO-DEST`
eg `kart clone C:\my-repos\example C:\my-repos\copy-of-example`
If you have ssh access to a machine, you can clone a repo from that machine to the one you are using with SSH:
eg `kart clone ssh://USERNAME@HOSTNAME/path-to-repo`
However, I think it's less common for windows users to have an SSH server set up, with it being a unix technology. An easier solution might be to set up a network drive you can access that contains the original repository, then clone it in the same way as you would clone a local one.

Kart cloning is so similar to Git cloning that most Git tutorials will be relevant, if that helps you.
Once you have some clones, it's possible to push and pull between them - this also works just like in Git, and also takes some getting used to. Let me know if you need to know more.

**Reply by Asare-Obed** [reply]
I have a similar challenge, but here is the case where I want to clone the repository on GitHub.
